### ISSUE TITLE
“Multi-line function call not indented correctly” for nested arrays

### ISSUE THREAD

**Issue by montchr** [WordPress-Coding-Standards/WordPress-Coding-Standards#494]
Since updating recently, I've been getting a ton of these errors for nested arrays and closures. Here's an example array:

```php
register_taxonomy( 'pedestal_source', [ 'pedestal_link' ], [
    'hierarchical'      => true,
    'public'            => true,
    'show_in_nav_menus' => true,
    'show_ui'           => true,
    'show_admin_column' => true,
    'query_var'         => true,
    'rewrite'           => true,
    'capabilities'      => [
        'manage_terms'  => 'edit_posts',
        'edit_terms'    => 'edit_posts',
        'delete_terms'  => 'edit_posts',
        'assign_terms'  => 'edit_posts',
    ],
    'labels'            => [
        'name'                       => __( 'Sources', 'pedestal' ),
        'singular_name'              => _x( 'Source', 'taxonomy general name', 'pedestal' ),
        'search_items'               => __( 'Search Sources', 'pedestal' ),
        'popular_items'              => __( 'Popular Sources', 'pedestal' ),
        'all_items'                  => __( 'All Sources', 'pedestal' ),
        'parent_item'                => __( 'Parent Source', 'pedestal' ),
        'parent_item_colon'          => __( 'Parent Source:', 'pedestal' ),
        'edit_item'                  => __( 'Edit Source', 'pedestal' ),
        'update_item'                => __( 'Update Source', 'pedestal' ),
        'add_new_item'               => __( 'New Source', 'pedestal' ),
        'new_item_name'              => __( 'New Source', 'pedestal' ),
        'separate_items_with_commas' => __( 'Sources separated by comma', 'pedestal' ),
        'add_or_remove_items'        => __( 'Add or remove Sources', 'pedestal' ),
        'choose_from_most_used'      => __( 'Choose from the most used Sources', 'pedestal' ),
        'menu_name'                  => __( 'Sources', 'pedestal' ),
    ],
] );
```

For each line in `capabilities` and `labels`, I receive the message:

> Multi-line function call not indented correctly;
> expected 12 spaces but found 16

**Comment by danielbachhuber** [comment]
I ran into this with https://github.com/WP-API/WP-API/pull/1764 (https://travis-ci.org/WP-API/WP-API/jobs/92832611)

I suspect it's coming from PHP Codesniffer, as WP-API is locked to WPCS 0.6

**Comment by montchr** [comment]
> I suspect it's coming from PHP Codesniffer, as WP-API is locked to WPCS 0.6

Yeah you're right. They just released version 2.4.0 [18 hours ago](https://github.com/squizlabs/PHP_CodeSniffer/releases/tag/2.4.0).

And the issue with short array syntax is already filed [over there](https://github.com/squizlabs/PHP_CodeSniffer/issues/786).

Closing this out.
